defaults/main.yml

```diff
@@ -1,5 +1,36 @@
 ---
-horizon_path: /opt/stack/horizon/
+panel_src_path_os:
```

Comment:
There are enough platform specific variables that I think it would be cleaner to setup individual files for them and then at the top of the main task do an include_vars with_first_found statement.